README.md

```diff
@@ -73,6 +73,10 @@ ansiColor('xterm') {
 }
 ```
 
+## Colormaps
+
+The argument to the ansiColor function can be either `term`, `vga`, `gnome-terminal` or `css`. This defines which colors Jenkins should show for some of the escape codes. You can find the exact colors used here: https://github.com/jenkinsci/ansicolor-plugin/blob/,aster/src/main/java/hudson/plugins/ansicolor/AnsiColorMap.java#L45.
```

Comment:
Let's make this a markdown link? `[here](link)`, and fix the `,aster` typo? You don't need to include github.... here either, these links are relative, so `src/main/java/hudson/plugins/ansicolor/AnsiColorMap.java` will do. I would probably exclude `#L45` because that file might change in the future, but up to you.

Reply:
Are these color maps some kind of standard? Maybe we can research that and include a link?

Reply:
Avoid linking the word "here". Hypertext links should add information to a word in context.

> This defines which set of colors Jenkins should show for some of the escape codes. The exact color values used in each set are defined in [AnsiColorMap.java](src/main/java/hudson/plugins/ansicolor/AnsiColorMap.java).